From 3e2ab9139b12b1239eebcbb6c6a23e7284dd7b44 Mon Sep 17 00:00:00 2001
From: Dan Nicholson <dbn.lists@gmail.com>
Date: Sat, 3 Apr 2010 09:33:51 -0700
Subject: [PATCH 10/25] Move 10-evdev.conf to system config dir $datadir/X11/xorg.conf.d

Since the server searches in a vendor specific path now, we can install
the evdev catchall there without disturbing local administration files.

Signed-off-by: Dan Nicholson <dbn.lists@gmail.com>
Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit d4dd3d0085f8eaca8c989aaeace1a1a24c5ebf49)

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 config/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git xorg-server-1.8.0/config/Makefile.am xorg-server-1.8.0/config/Makefile.am
index eed7454..675a3b2 100644
--- xorg-server-1.8.0/config/Makefile.am
+++ xorg-server-1.8.0/config/Makefile.am
@@ -9,7 +9,7 @@ AM_CFLAGS += $(UDEV_CFLAGS)
 libconfig_la_SOURCES += udev.c
 libconfig_la_LIBADD = $(UDEV_LIBS)
 
-xorgconfddir = $(prefix)/etc/X11/$(XF86CONFIGDIR)
+xorgconfddir = $(datadir)/X11/$(XF86CONFIGDIR)
 xorgconfd_DATA = 10-evdev.conf
 
 else
-- 
1.7.0.4

